### PR TITLE
Modify home page UI and navigation links

### PR DESCRIPTION
**Pull request by geovgy**
To the REVAMP branch. This PR includes cleaning up the home landing screen of the dapp and fix the href links in the header and footer. A very small PR that doesn't affect anything else in the dapp.

### CODE REVIEW

File: packages/dapp/src/pages/Home.jsx [hunk 1]
```diff
@@ -51,27 +52,62 @@ export const Home = () => {
...
+          // size={buttonSize}
+          // width='fit-content'
           fontFamily="mono"
           fontWeight="normal"
         >
-          {smallScreen ? 'CREATE NEW INVOICE' : 'CREATE A NEW SMART INVOICE'}
+          CREATE NEW INVOICE
         </Button>
         <Text
           fontWeight="bold"
```
Comment (plor) [inline packages/dapp/src/pages/Home.jsx:113]: Not a fan of commented out code, the next comment is going to be on the remainder of it 😄

File: packages/dapp/src/pages/Home.jsx [hunk 2]
```diff
@@ -83,13 +119,16 @@ export const Home = () => {
...
           onClick={viewInvoices}
           size={buttonSize}
           fontFamily="mono"
           fontWeight="normal"
         >
-          VIEW EXISTING INVOICE
+          VIEW EXISTING INVOICES
         </Button>
         <Button
           mt="2rem"
```
Comment (plor) [inline packages/dapp/src/pages/Home.jsx:134]: Github is making me leave three comments to remark on the commented out code.

File: packages/dapp/src/pages/Home.jsx [hunk 3]
```diff
@@ -120,7 +159,7 @@ export const Home = () => {
             })}
         py="3rem"
         m="1rem"
-      ></Flex>
-    </Container>
+      ></Flex> */}
```
Comment (plor) [inline packages/dapp/src/pages/Home.jsx:162]: Here's the last one. Not a big deal, I just prefer removing and going into history to reclaim old code.

**Review by plor — APPROVED** [review]

File: packages/dapp/src/pages/Home.jsx (hunk 3 above)
Comment (geovgy) [reply packages/dapp/src/pages/Home.jsx:162]: That's fair. I'll clean it up and remove the commented out sections. Forgot about them.

**Pull request merged**